api/resolvers/serial.js

```diff
@@ -85,8 +85,9 @@ export async function serializeInvoicable (query, { models, lnd, hash, hmac, me,
   let results = await serialize(models, ...trx)
 
   if (hash) {
-    if (invoice?.isHeld) await settleHodlInvoice({ secret: invoice.preimage, lnd })
-    results = results.slice(1, -1)
+    if (invoice?.isHeld) { await settleHodlInvoice({ secret: invoice.preimage, lnd }) }
+    // remove first element since that is the confirmed invoice
+    [, ...results] = results
```

Comment:
I actually don't know what the first element is but it should be the confirmed invoice. But it's a 1, not a 0 which is what `confirm_invoice` should return. :thinking: